src/components/ble/NavigationService.h

```diff
@@ -27,15 +27,11 @@
 #undef min
 
 namespace Pinetime {
-  namespace System {
-    class SystemTask;
-  }
-
   namespace Controllers {
 
     class NavigationService {
     public:
-      explicit NavigationService(Pinetime::System::SystemTask& system);
+      explicit NavigationService();
```

Comment:
The `explicit` specifier isn't necessary here anymore. It was necessary because the constructor had 1 argument to prevent it from being used for implicit type conversions.